frontend/src/component/project/Project/ProjectInsights/FlagTypesUsed/FlagTypesUsed.tsx

```diff
@@ -0,0 +1,125 @@
+import { useMemo } from 'react';
+import { styled, type SvgIconTypeMap, Typography } from '@mui/material';
+import { getFeatureTypeIcons } from 'utils/getFeatureTypeIcons';
+
+import type { OverridableComponent } from '@mui/material/OverridableComponent';
+import { useRequiredPathParam } from 'hooks/useRequiredPathParam';
+import useProjectOverview from 'hooks/api/getters/useProjectOverview/useProjectOverview';
+
+export const StyledProjectInfoWidgetContainer = styled('div')(({ theme }) => ({
+    margin: '0',
+    [theme.breakpoints.down('md')]: {
+        display: 'flex',
+        flexDirection: 'column',
+        position: 'relative',
+    },
+}));
+
+export const StyledWidgetTitle = styled(Typography)(({ theme }) => ({
+    marginBottom: theme.spacing(2.5),
+}));
+
+export const StyledCount = styled('span')(({ theme }) => ({
+    fontSize: theme.typography.h2.fontSize,
+    fontWeight: 'bold',
+    color: theme.palette.text.primary,
+}));
+
+const StyledTypeCount = styled(StyledCount)(({ theme }) => ({
+    marginLeft: 'auto',
+    fontWeight: theme.typography.fontWeightRegular,
+    color: theme.palette.text.secondary,
+}));
+
+interface IFlagTypeRowProps {
+    type: string;
+    Icon: OverridableComponent<SvgIconTypeMap>;
```

Comment:
I know it was probably before but it looks super weird. Usually this type of thing is handled with children